### DISCUSSION TITLE
installing nodejs dependencies failure

### DISCUSSION THREAD

**Discussion by surendraalamuru22** [learndevopsonline/learndevopsonline#1888] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/38e546eb-8bd3-46e2-8421-dc8d8848cd87)

**Comment by chaithanya1812** [comment]
Hi @surendraalamuru22 ,
## Isuue:- check the both directory paths are same??

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/ebf0e676-bee7-4144-b1a3-e3d18c352e26)

**Reply by surendraalamuru22** [reply]
i didn't understood chaitanya

**Reply by chaithanya1812** [reply]
Hi @surendraalamuru22 

Isuue:- check the both directory paths are same?? 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/85a951d2-0114-48a6-a288-b88ef0d4957e)

## Note: still you can't understand, Join the Lab-session

**Reply by surendraalamuru22** [reply]
ok chaitu
